data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -107,13 +149,13 @@ _ "This attack uses 6 attacks"#enddef
     [event]
         name = new turn
         first_time_only = no
-        id = song_verse_new_turn_{X}
+        id = song_verse_new_turn_{VERSE}
 
         [modify_unit]
             [filter]
                 side = $side_number
                 [has_attack]
-                    special_id = verse_{DISTINGUISHER}
+                    special_id = verse_{VERSE}
```

Comment:
If I'm understanding this code correctly, an id such as `song_with_{MAX}_verses` would be clearer.

Reply:
That sounds like applying to unit more than to the attacks. IMO makes sense if filtering attacks.

Reply:
The main issue is `VERSE` vs `MAX`. This isn't the verse number, as the specials all 3 of the Sun Sylph's attacks have `id = verse_3`:

Issue #7565 makes it harder to check, but `:inspect`, select a Sylph, and copy her to the clipboard.
```
			description=_"This attack is the first verse of ..."
			id="verse_3"
			name=_"first verse"
```